tests/Install-Scripts.Test/AkaMsLinksTests.cs

```diff
@@ -277,6 +277,85 @@ public void LinkCanBeCreatedForGivenRuntime(string channel, string runtime, stri
             commandResult.Should().HaveStdOutContaining(output => Regex.IsMatch(output, expectedLinkPattern));
         }
 
+        [Theory]
+        [InlineData("3.1", null, @"https://aka.ms/dotnet/internal/3.1/dotnet-sdk-")]
+        [InlineData("5.0.2xx", null, @"https://aka.ms/dotnet/internal/5.0.2xx/dotnet-sdk-")]
+        [InlineData("Current", null, @"https://aka.ms/dotnet/internal/current/dotnet-sdk-")]
+        [InlineData("LTS", null, @"https://aka.ms/dotnet/internal/LTS/dotnet-sdk-")]
+        [InlineData("5.0.2xx", "validated", @"https://aka.ms/dotnet/internal/5.0.2xx/validated/dotnet-sdk-")]
+        public void InternalLinkCanBeCreatedForSdk(string channel, string quality, string expectedLink)
```

Comment:
consider consolidating content from both tests, they share a lot of the same exact code.

Reply:
I discussed this with @bozturkMSFT and the current structure is that we keep use cases for sdk and for runtime separate in unit tests. @bozturkMSFT, could you please comment on the reasons for that?

Reply:
I'd say that we do not want to have too many ifs in the unit test (checking whether we have runtime specified in the command), and do not want to confuse ourselves with different parameters that we pass to the test and their possible combinations. Maybe there are also other reasons for that.

Reply:
The reason is not to complicate the logic in the test methods.